free-courses-hi.md

```diff
@@ -1,46 +1,76 @@
 
 
 ### C
 
-* [All C Concepts | Hindi](https://www.youtube.com/playlist?list=PL7ersPsTyYt1d8g5qaxbE6sjWDzs4D_1v) - Saurabh Shukla
+* [All C Concepts | Hindi](https://www.youtube.com/playlist?list=PL7ersPsTyYt1d8g5qaxbE6sjWDzs4D_1v) - Saurabh Shukla (Youtube)
+* [C Language Tutorials In Hindi](https://www.youtube.com/playlist?list=PLu0W_9lII9aiXlHcLx-mDH1Qul38wD3aR) - Harry (Youtube)
 
 
 ### C++
 
-* [The Complete C++ Course | Hindi](https://www.youtube.com/playlist?list=PLLYz8uHU480j37APNXBdPz7YzAi4XlQUF) - Saurabh Shukla
+* [The Complete C++ Course | Hindi](https://www.youtube.com/playlist?list=PLLYz8uHU480j37APNXBdPz7YzAi4XlQUF) - Saurabh Shukla (Youtube)
+* [C++ Tutorials In Hindi](https://www.youtube.com/playlist?list=PLu0W_9lII9agpFUAlPFe_VNSlXW5uE0YL) - Harry (Youtube)
+
+
+### Git
+
+* [Complete Git Tutorials For Beginners In Hindi](https://www.youtube.com/playlist?list=PLu0W_9lII9agwhy658ZPA0MTStKUJTWPi) - Harry (Youtube)
+* [Git and GitHub Tutorials In Hindi](https://www.youtube.com/playlist?list=PLu0W_9lII9ahVQekD7ePHmnirTePXwIln) - Harry (Youtube)
 
 
-### Data Structures
+### HTML / CSS
 
-* [All Data Structure Concepts | Hindi](https://www.youtube.com/playlist?list=PLsFNQxKNzefJNztGGoQC-59UhSwIaiIW3) - Saurabh Shukla
-* [All Sorting algorithms and Programs | Hindi](https://www.youtube.com/playlist?list=PLsFNQxKNzefJU-Sj__mljvrmJHZVKWbEm) - Saurabh Shukla
-* [Data Structure Programs | Hindi](https://www.youtube.com/playlist?list=PLsFNQxKNzefK_DAUwnQwBizOmcY7aDLoY) - Saurabh Shukla
+* [HTML and HTML5 (Hindi)](https://www.youtube.com/playlist?list=PLbGui_ZYuhihSr4HroBBQt_quHL42JLif) - Geeky Shows (Youtube)
+* [CSS and CSS 3 (Hindi)](https://www.youtube.com/playlist?list=PLbGui_ZYuhiisW-k9eEON2vxGABv-PafB) - Geeky Shows  (Youtube)
 
 
 ### Java
 
-* [The complete Java tutorial | Hindi](https://www.youtube.com/playlist?list=PLu0W_9lII9agS67Uits0UnJyrYiXhDS6q) - Harry
+* [The complete Java tutorial | Hindi](https://www.youtube.com/playlist?list=PLu0W_9lII9agS67Uits0UnJyrYiXhDS6q) - Harry (Youtube)
 
 
 ### Javascript
 
-* [JavaScript Tutorial | Hindi](https://www.youtube.com/playlist?list=PLu0W_9lII9ajyk081To1Cbt2eI5913SsL) - Harry
+* [JavaScript Tutorial | Hindi](https://www.youtube.com/playlist?list=PLu0W_9lII9ajyk081To1Cbt2eI5913SsL) - Harry (Youtube)
+
+
+### .NET
+
+* [.NET Framework (Hindi)](https://www.youtube.com/playlist?list=PLbGui_ZYuhij4hXs7Lt93tfv4nA1BoYdb) - Geeky Shows (Youtube)
 
 
 ### Python
 
-* [Python Tutorial For Beginners | Hindi (With Notes)](https://www.youtube.com/playlist?list=PLu0W_9lII9agICnT8t4iYVSZ3eykIAOME) - Harry
+* [Python Tutorial For Beginners | Hindi (With Notes)](https://www.youtube.com/playlist?list=PLu0W_9lII9agICnT8t4iYVSZ3eykIAOME) - Harry (Youtube)
+* [Intermediate/Advanced python Tutorials in Hindi](https://www.youtube.com/playlist?list=PLu0W_9lII9aiJWQ7VhY712fuimEpQZYp4) - Harry (Youtube)
+
+
+### Django
```

Comment:
This is a subsection under Python, could you give it h4 instead of h3?

```suggestion
#### Django
```

Reply:
Removed outdated course & Made h4 heading to "Django".